projects/rocprofiler-systems/source/lib/core/trace_cache/metadata_registry.cpp

```diff
@@ -278,6 +278,89 @@ metadata_registry::get_kernel_symbol_list() const
     return result;
 }
 
+void
+metadata_registry::overwrite_cb_names(
+    std::initializer_list<
+        std::pair<rocprofiler_callback_tracing_kind_t, std::map<int, std::string_view>>>
+        categories_info)
+{
+    if(categories_info.size() == 0) return;
+
+    using category   = rocprofiler_callback_tracing_kind_t;
+    using operations = std::vector<std::string_view>;
```

Comment:
```suggestion
    using operation_names_t = std::vector<std::string_view>;
```
Using a different type naming convention so it's not mistaken as a variable.